.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
```

Comment:
Please add a comment about the root thing here. IMO this is a little confusing. Something similar which is used in the official docs should be good:
```suggestion
# top-most EditorConfig file
root = true
```